### ISSUE TITLE
How to use two steps on same element?

### ISSUE THREAD

**Issue by stefaneidelloth** [kamranahmedse/driver.js#353]
When using following code only the first step is shown.
How to fix this?

```
const driver = new Driver();
driver.defineSteps([
  {
    element: '#my-id',
    popover: {                         
      title: 'First title',
      description: 'First description'                         
    }
  },
  {
    element: '#my-id',
    popover: {
      title: 'Second title',
      description: 'Second description'                         
    }
  }
]);
driver.start();
```

**Comment by tector** [comment]
related to #310

**Comment by kamranahmedse** [comment]
We just released [v1.0](https://github.com/kamranahmedse/driver.js/releases/tag/1.0.3) of driver.js which has been rewritten from the ground up. It allows you to have same element multiple times in the tour. Please have a [look at the docs](https://driverjs.com/) and do let me know if you face any issues.
